_dashboards/dashboard/index.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: default
+title: Creating dashboards
+nav_order: 30
+has_children: true
+---
+
+# Creating dashboards
+
+A dashboard combines multiple data visualizations into a single view. With the **Dashboard** application, you can:
+
+- Get started with pre-built panels and dashboards and customize display properties to create the dashboard you need.
+- Analyze, monitor, and alarm on metrics, logs, and traces across multiple data sources.
+- Create interactive dashboards and share them with anyone in your organization.
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/dashboard-index.png" alt="Single-view dashboard in OpenSearch Dashboards" width="600" height="300">
+
+Before you begin this tutorial, make sure you've installed OpenSearch and OpenSearch Dashboards and that you've connected to Dashboards at [http://localhost:5601](http://localhost:5601). The username and password are `admin`.
+{: .note}
+
+## Getting familiar with user interface
+
+In this tutorial you'll learn the basics of creating a dashboard using the **Dashboard** application and OpenSearch sample data. Before getting started, get familiar with the Dashboard user interface. The user interface comprises five main parts, as shown in the following image:
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/user-interface.png" alt="Dashboard user interface" width="600" height="300">
+
+- The **navigation panel** (A) on the left contains the OpenSearch Dashboards applications.
+- The **toolbar** (B) contains frequently used commands and shortcuts.
+- The **search** bar (C) allows you to search for documents and other objects and add filters.
+- The **time filter** (D) allows you to customize the time and date.
+- The **panel** (E) allows you to add existing visualizations to the dashboard or create new ones for the dashboard.
+
+
+_Panel_ is a term used to refer to a visualization displayed on a dashboard. The terms _panel_ and _visualization_ may be used interchangeably throughout this and other Dashboard-related documentation.
+{: .note}
+
+## Adding sample data
+
+If you haven't already added the OpenSearch Dashboards sample datasets as described in [Quickstart guide for OpenSearch Dashboards](https://opensearch.org/docs/latest/dashboards/quickstart-dashboards/), add them before proceeding to the next steps. **Sample eCommerce orders** is used for this tutorial. 
```

Comment:
Out of scope - we should create an issue about making re-usable snippets for repeated content like this. As we add more pages, it will be increasingly difficult to keep all these copy/pasted sections up to date and in sync.

Reply:
Good point. Deleted.